src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,95 @@
-export const Pagination = () => {};
+import { useEffect } from 'react';
+
+interface Props {
```

Comment:
The 'pages' prop is not defined in the task. It seems like this prop is used to store the possible pages. The component should calculate this dynamically based on the 'total', 'perPage' and 'currentPage' props.